packages/kbn-optimizer/src/optimizer/optimizer_config.test.ts

```diff
@@ -98,7 +98,7 @@ describe('OptimizerConfig::parseOptions()', () => {
           <absolute path>/src/plugins,
           <absolute path>/x-pack/plugins,
           <absolute path>/plugins,
-          <absolute path>-extra,
+          "/Users/angelachuang/workspace/kibana-extra",
```

Comment:
we probably want to keep these as templates, can this change be backed out?

Reply:
ahh, absolutely, thank you!